servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ConfigValidateTestBase.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.azure.servicebus;
```

Comment:
Add license header. And you have incorrect spacing you can fix with Ctrl+Alt+L (Reformat code). https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf

Reply:
updated